src/page/Profile/Profile.jsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { IoSettingsOutline } from "react-icons/io5";
+import { PostCard } from "../../components/componentExport";
+
+export const Profile = () => {
+  return (
+    <div className="w-screen h-screen">
+      <div className="flex flex-col justify-center items-center">
+        <div
+          className="w-full max-w-xl m-auto border-b-2 flex justify-start
```

Comment:
instead of writing multiple classNames, it would be better we can write in a single className

Reply:
ya, it's good idea, I also feel to do so. thanks